packages/pds/src/api/app/bsky/feed/getTimeline.ts

```diff
@@ -27,18 +27,27 @@ export default function (server: Server, ctx: AppContext) {
 
       const postsQb = feedService
         .selectPostQb()
-        .where('creator', '=', requester)
-        .orWhere('creator', 'in', followingIdsSubquery)
+        .where((qb) =>
```

Comment:
just curios, what is the benefit of this syntax?

Reply:
Good q— that puts parens around the sql clause.  Without the parens, the "or" condition can match and cause a result to be included even if a required condition within `selectPostQb()` is not matched.

Reply:
ah got it got it. thx 👌